.circleci/config.yml

```diff
@@ -75,6 +75,16 @@ jobs:
           command: cargo test
           working_directory: ./primitives
           no_output_timeout: 60m
+      - run:
+          name: mips tests
+          command: cargo test
+          working_directory: ./pallets/mips
+          no_output_timeout: 60m
+      - run:
+          name: transaction-payment tests
+          command: cargo test
+          working_directory: ./pallets/transaction-payment
```

Comment:
We should add these directories to caching directories as well for CI performance. Ps this can be done in new a PR.